x/ethbridge/types/errors.go

```diff
@@ -27,3 +28,7 @@ func ErrInvalidEthAddress(codespace sdk.CodespaceType) sdk.Error {
 func ErrJSONMarshalling(codespace sdk.CodespaceType) sdk.Error {
 	return sdk.NewError(codespace, CodeErrJSONMarshalling, "error marshalling JSON for this claim")
 }
+
+func ErrInvalidEthSymbol(codespace sdk.CodespaceType) sdk.Error {
+	return sdk.NewError(codespace, CodeInvalidEthSymbol, "invalid symbol provided, symbol \"eth\" must have null address set as token contract address")

```

Comment:
should the error be `ErrInvalidEthAddress` then based on this description?

Reply:
@fedekunze just to confirm - are you recommending that we use the existing `ErrInvalidEthAddress` error for the case when the symbol does not match the address, or that we should rename the `ErrInvalidEthSymbol` error?

Reply:
either is fine tbh

Reply:
https://github.com/cosmos/peggy/pull/68/commits/a240c3338526d1d56f01fd32acb4b168343834a7